server/src/repos/SettingsRepo.ts

```diff
@@ -1,11 +1,21 @@
-import {Path, readFile, writeFile} from "jsonfile";
-import {ISettings} from "@src/models/Settings";
+import * as fs from 'fs/promises' // Moved away from node-jsonfile since it seemed tiny and does not have code coverage reports
+import { ISettings } from '@src/models/Settings'
+import { getSystemErrorName } from 'util'
 
 export async function readSettingsJson(): Promise<ISettings> {
 	// Paths are relative to the directory the server is running out of
-	return await readFile("./settings/settings.json");
+	try {
+		return JSON.parse(await fs.readFile('./settings/settings.json', 'utf-8'))
+	} catch (e: any) {
+		if (getSystemErrorName(e.errno) === 'ENOENT') {
```

Comment:
I was thinking that the program is going to save the new settings whenever the user changes the default settings anyway. However, we could change the output so that the parent function can see whether it opened the example and warn the user that changes need updating.